Local/Factoids.py

```diff
@@ -31,6 +31,7 @@ def handle(self, message):
             txt = 'IRC is not Twitter – please do not use @name to address people. use name,' \
             ' or name: instead. Using @ also prevents you from tab-completing nicks in your IRC client'
         elif command == '!source':
+            txt = "You can find me on: 'https://github.com/voxpupuli/thevoxfox'"
```

Comment:
Looks like some weirdness happening here. I see `txt ==` on 35 and `txt =` on 34.